src/components/TableComponents/PreHeaderBox/PreHeaderBox.tsx

```diff
@@ -0,0 +1,34 @@
+import { ReactComponent as FilterIcon } from "../../../assets/logos/filter.svg";
+import { ReactComponent as SortIcon } from "../../../assets/logos/sort.svg";
+
+import {
+  PreHeaderBoxWrapper,
+  SortAndFilterBox,
+  SortAndFilterItem,
+  SortAndFilterItemText,
+  AddContactText,
+} from "./styled";
+
+interface IPreHeaderBoxProps {
+  handleOpen: () => void;
+}
+
+const PreHeaderBox = ({ handleOpen }: IPreHeaderBoxProps) => {
+  return (
+    <PreHeaderBoxWrapper>
+      <SortAndFilterBox>
+        <SortAndFilterItem>
+          <SortIcon />
+          <SortAndFilterItemText>Sort</SortAndFilterItemText>
+        </SortAndFilterItem>
+        <SortAndFilterItem>
+          <FilterIcon />
+          <SortAndFilterItemText>Filter</SortAndFilterItemText>
+        </SortAndFilterItem>
+      </SortAndFilterBox>
+      <AddContactText onClick={handleOpen}>+ &nbsp;Add contact</AddContactText>
```

Comment:
Its shared component, but not universal. `AddContactText`? What if it will be not a Contact but a Ticket? or Task? Ot whatever?
It should be configurable with props.

Reply:
Solved, thx.